packages/TelegramClient-Tests.package/TCTUITest.class/instance/setUp.st

```diff
@@ -0,0 +1,8 @@
+running
+setUp
+
+	| core | 
+	core := TCTMockCore basicNew initializeHandlers.
+	core chats: TCTMocks mockChats. 
+	self app: (TCUMain newWithCore: core). 
+	self wantsToTest: self app. 
```

Comment:
It is a bit weird, that TCTUITest has test cases in the first place. Maybe we should rename to TCTUITestCase and only place the test cases in subclasses? We placed all of ours into TCTAcceptanceTests.

How about you put your method into the subclass? Your mock core should also already work for `testCanOpenChatInfoPage` and `testCanCloseChatInfoPage` as long as some group chats are available. I think making them use the subject you create instead of creating their own with a mock core would be great.

The only issue is having both those tests and the testMessageAuthors in the same class, when they need different things for setup. I think the user-store MR could solve that though, right @phkeese ? (Especially when the NoTDLibCore and the MockCore are merged into the same thing, then creating one of those in the setUp of TCTAcceptanceTests would be all we ever need).

Reply:
we will fix this in #370